packages/bruno-lang/v2/src/bruToJson.js

```diff
@@ -105,7 +105,7 @@ const mapPairListToKeyValPairs = (pairList = [], parseEnabled = true) => {
   }
   return _.map(pairList[0], (pair) => {
     let name = _.keys(pair)[0];
-    let value = decodeURIComponent(pair[name]);
+    let value = name === 'password' ? pair[name] : decodeURIComponent(pair[name]);
```

Comment:
@bpoulaindev We need to solve this in a generic way and should not hardcode this to apply to any key-val pair that has the name `password`

The `mapPairListToKeyValPairs` is used to parse all key val pairs (headers, assertions, vars, query params)

Reply:
@helloanoop how could you have a generic function that work both on a password (which can be of any form), a random string and an encoded URL with params ?